com.unity.render-pipelines.universal/Shaders/BakedLit.shader

```diff
@@ -273,15 +282,25 @@ Shader "Universal Render Pipeline/Baked Lit"
 
             #pragma vertex vert
             #pragma fragment frag
-            #pragma shader_feature_local _ _NORMALMAP
+
+            // -------------------------------------
+            // Material Keywords
+            #pragma shader_feature_local _NORMALMAP
             #pragma shader_feature_local_fragment _ALPHATEST_ON
             #pragma shader_feature_local_fragment _ALPHAPREMULTIPLY_ON
 
+            // -------------------------------------
+            // Universal Pipeline keywords
+            #pragma multi_compile_fragment _ _SCREEN_SPACE_OCCLUSION
```

Comment:
This was missing for SM2.0 Baked lit shader.

Reply:
I'm not sure if still the case, it used to be that shader_feature didn't need the _ while multi_compile did.

Reply:
I'd actually prefer we always have _ as the first parameter when applicable. I want to create a PR adding those.

When you have a single keyword like:
`#pragma shader_feature _OPTION`
We will create the `_` or `_OPTION_OFF` variant automatically, which is the default state. However, if you have 2 or more keywords we don't so you need to have the "OFF" variant as well. Example the Detail:
`#pragma shader_feature_local _ _DETAIL_MULX2 _DETAIL_SCALED`

Reply:
I agree with elvar on always having _, IIRC only shader_feature has it. It creates this tribal knoweldge thingy and also very easy to get in trouble in the example he showed.